pkgs/servers/pihole-adminlte/default.nix

```
@@ -0,0 +1,54 @@
+{ stdenv, lib, fetchFromGitHub, php, pihole, pihole-ftl, procps, ... }:
+
+stdenv.mkDerivation rec {
+  pname = "pihole-adminlte";
+  version = "5.19";
+
+  src = fetchFromGitHub {
+    owner = "pi-hole";
+    repo = "AdminLTE";
+    rev = "v${version}";
+    sha256 = "sha256-Sq/51cdSO3NSy8Cw9mgNDDTD60vAH0jUcSAswO59ryM=";
+  };
+
+  propagatedBuildInputs = [
+    pihole
+    pihole-ftl
+    procps
+  ];
+
+  postPatch = ''
+    # Redirect stderr to stdout when running gravity on the Update Gravity page
+    substituteInPlace scripts/pi-hole/php/gravity.sh.php \
+      --replace 'sudo pihole -g' '${pihole}/bin/pihole -g 2>&1'
+
+    substituteInPlace settings.php \
+      --replace '/usr/bin/pihole-FTL' ${pihole-ftl}/bin/pihole-FTL
+
+    substituteInPlace scripts/pi-hole/php/func.php \
+      --replace 'pidof pihole-FTL' '${procps}/bin/pidof pihole-FTL' \
+      --replace "ps -p" "${procps}/bin/ps -p"
+
+    substituteInPlace scripts/pi-hole/php/func.php \
+      --replace 'sudo pihole' ${pihole}/bin/pihole
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share
+    cp -r -t $out/share *.php img/ scripts/ style/
+
+    mkdir -p $out/share/doc/$name/
+    cp README.md $out/share/doc/$name/
```

```suggestion
    mkdir -p $out/share/doc/pihole-adminlte/
    cp README.md $out/share/doc/pihole-adminlte/
```
normally we don't install those but whatever